kubernetes_tests/test_kubernetes_executor.py

```diff
@@ -20,16 +20,18 @@
 import unittest
```

Comment:
Why we rename this test file?

Reply:
We moved it. Now all tests that require KinD cluster are not run in CI image. They are run externally in virtualenv created on host and the core "airflow" there is deployed to the Kind Cluster in the form of slightly modified Production Image. 

The unit tests there are literally just "clients" to run the actual Pods and Dags in the Airflow installed in the cluster so we do not need the whole CI environment for those tests (and makes it also faster and easier to run on the CI. 

This way you only need to run `./breeze kind-cluster deploy` to deploy Airflow to Kind Cluster (using production image as a base). I tried to describe it in the TESTING.rst but I will add some more information and possibly a chart showing the architecture.

Reply:
Hm, still not sure if we should keep tests in two separate directories. On the other hand, I know projects that have e2e tests separately so it can make sense...

Reply:
Yeah. I thouhgt about using markers, but I think those are very special kind of tests -not even system tests. They literally expect Airflow/Kubernetes to be running elsewhere.